.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "printWidth": 100
```

Comment:
Is there a reason for using 100 over the default 80?

Reply:
Yeah I set it up to closer match the initial formatting

Reply:
I think I already had it prettier'd on save with my global settings which has it at 80. I see some arrays have changed formatting — can we just leave it at 80 / default settings?